From 566e693a84ead2cb670ef44d7b10d7dad7240bdf Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:07:10 -0400
Subject: [powerpc] boot: Ignore .dtb files

Message-id: <1410545655-205645-202-git-send-email-dzickus@redhat.com>
Patchwork-id: 94661
O-Subject: [RHEL7 PATCH 201/626] powerpc/boot: Ignore .dtb files.
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 2abe3b8e73df9895b89f907b7a93d7b800427583
Author: Ian Campbell <Ian.Campbell@citrix.com>
Date:   Mon Jun 3 12:00:56 2013 +0100

    powerpc/boot: Ignore .dtb files.

    Signed-off-by: Ian Campbell <ian.campbell@citrix.com>
    Cc: Benjamin Herrenschmidt <benh@kernel.crashing.org>
    Cc: Paul Mackerras <paulus@samba.org>
    Cc: linuxppc-dev@lists.ozlabs.org
    Cc: linux-kernel@vger.kernel.org
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/boot/.gitignore b/arch/powerpc/boot/.gitignore
index c32ae5c..8fa258e 100644
--- a/arch/powerpc/boot/.gitignore
+++ b/arch/powerpc/boot/.gitignore
@@ -16,6 +16,7 @@ mktree
 uImage
 cuImage.*
 dtbImage.*
+*.dtb
 treeImage.*
 zImage
 zImage.initrd
-- 
1.7.1